Makefile

```diff
@@ -292,38 +283,59 @@ $(MLSTDB_DIR)/blast:
 	cd $(MLSTDB_DIR) \
 	&& apptainer exec \
 		--bind $(MNT_ROOT) \
-		$(CONTAINER_DIR)/blast.sif \
+		$(CONTAINERS_DIR)/blast.sif \
 		bash $(MLSTDB_DIR)/mlst-make_blast_db.sh |& tee -a $(INSTALL_LOG)
 
 # -----------------------------
-# Update VirulenceFinder database
+# Update Finder databases
 # -----------------------------
+
+update_finder_dbs: update_virulencefinder_db \
+	update_resfinder_db \
+	update_pointfinder_db \
+	update_serotypefinder_db
+
 VIRULENCEFINDERDB_DIR := $(ASSETS_DIR)/virulencefinder_db
-KMA_DIR := $(ASSETS_DIR)/kma
+RESFINDERDB_DIR := $(ASSETS_DIR)/resfinder_db
+POINTFINDERDB_DIR := $(ASSETS_DIR)/pointfinder_db
+SEROTYPEDFINDERDB_DIR := $(ASSETS_DIR)/serotypefinder_db
 
-update_finder_dbs: $(VIRULENCEFINDERDB_DIR)/stx.name
 
-$(ASSETS_DIR)/virulencefinder_db/stx.name: | check-and-reinit-git-submodules
+update_virulencefinder_db: $(VIRULENCEFINDERDB_DIR)/s.aureus_hostimm.length.b
```

Comment:
I wonder if you meant to use `: |` instead of just `:` here, to make it clear that the existence of this file is enough, instead of also the timestamp?

(See explanation in the top of the Makefile)

Reply:
See above comment